src/System.Linq.Expressions/src/System.Linq.Expressions.csproj

```diff
@@ -9,7 +9,8 @@
     <AssemblyName>System.Linq.Expressions</AssemblyName>
     <RootNamespace>System.Linq.Expressions</RootNamespace>
     <AssemblyVersion>4.0.11.0</AssemblyVersion>
-    <DefineConstants Condition=" '$(IsInterpreting)' != 'true' ">$(DefineConstants);FEATURE_CORECLR</DefineConstants>
+    <DefineConstants Condition=" '$(IsInterpreting)' != 'true' ">$(DefineConstants);FEATURE_COMPILE</DefineConstants>
+    <DefineConstants Condition=" '$(FeatureInterpret)' == 'true' ">$(DefineConstants);FEATURE_INTERPRET</DefineConstants>
```

Comment:
Still need to make this change, if it's ok for the "preferInterpretation" overload to Compile to show up as a public API when IsInterpreting is set to true (if which case, should it be there all the time, so users that start off on a platform with only the interpreter available don't see that overload go missing when moving to a platform that historically only had the compiler)?